cvss2/score.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cvss2
+
+import (
+	"fmt"
+	"math"
+)
+
+func roundTo1Decimal(x float64) float64 {
+	// round up to one decimal
```

Comment:
Feels like it belongs to above the function definition. Frankly, considering function's name, I'd dropped it whatsoever. A total nitpick, tho.

Reply:
dropped